data/human/transport missions.txt

```diff
@@ -3209,3 +3209,97 @@ mission "Earth Retirement"
 			label reject
 			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the Paradise Worlds."`
 			`	You wish the couple the best of luck on Earth, and they slowly make their way to the train station. Presumably, their new apartment is far from here.`
+
+
+
+mission "Spacediving professionals"
+	name "Spacediving at <planet>"
+	description "Head to destination with <bunks> skydivers, then drop them from the edge of space as you land. They are wearing thermal spacesuits, durable parachutes and oxygen bottles to ensure their survival. Extreme sports like skydiving typically carry more risks than regular passenger transports. Payment is <payment>."
```

Comment:
Should this be `Head to <destination>`? I notice that none of these descriptions actually use a destination replacement. If the `<destination>` text replacement is too long, then I think that using `<planet>` would be more appropriate than not mentioning the actual destination at all in the description, or say "the destination" instead of just "destination."